mantidproject/mantid
# Muon analysis and results table

The muon analysis interface had a bug that prevented results table being generated correctly. 

Fixes #22153. 

**To test:**

Open muon analysis
Go to the data analysis tab and make sure multiple fitting is unticked
On the home tab load some MUSR 62260
Go to the data analysis tab and add a fitting function (doesn't matter which)
Do a fit

Then click sequential fit. 
In the runs box add a 4 (so it reads 62260-4)
Click start
When its complete close the box

Go to the home tab and load run MUSR 62270
Go to data analysis and do a fit

Go to the home tab and load run MUSR 62271
Go to data analysis and do a fit

Go to the settings tab and tick multiple fitting

In data analysis add a fitting function 
Click simultaneous
In the runs box add a -3 (reads 62271-3)
Do a fit

Go to the results table  tab
Click create table
It should have data for 62260, 62270 and 62271

Click the sequential  button and then create a table
 It should contain data for runs 62260, 62261, 62262,.  62263, 62264

Click simultaneous then create table
Should make a table

Click multiple then create table
Should make a table

Go to data anlysis
Go to fit -> sequential fit
In the runs box type have 62271, 62280-3
Go to results table
Click sequential and create table
The table should contain data for runs 62271, 62280 ,62281, 62282, 62283
**Release Notes** 

In release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
